applications/IgaApplication/iga_application.cpp

```diff
@@ -34,6 +36,7 @@ void KratosIgaApplication::Register() {
 
     // ELEMENTS
     KRATOS_REGISTER_ELEMENT("Shell3pElement", mShell3pElement)
+    KRATOS_REGISTER_ELEMENT("MembraneElement", mMembraneElement)
```

Comment:
Can you please rename this element to e.g. IgaMembraneElement. The problem ist that otherwise we would run into conflicts with the structural mechanics application.